cli/azd/internal/repository/infra_confirm.go

```diff
@@ -111,6 +112,24 @@ func (i *Initializer) infraSpecFromDetect(
 					AuthUsingUsernamePassword: authType == scaffold.AuthType_PASSWORD,
 				}
 				break dbPrompt
+			case appdetect.DbCosmos:
+				if dbName == "" {
+					i.console.Message(ctx, "Database name is required.")
+					continue
+				}
+				containers, err := detectCosmosSqlDatabaseContainersInDirectory(detect.root)
+				if err != nil {
+					return scaffold.InfraSpec{}, err
+				}
+				spec.DbCosmos = &scaffold.DatabaseCosmosAccount{
+					// todo:
+					// Now all services (except aca) are named by '${abbrs.xxx}${resourceToken}'
+					// Consider to name it by AccountName defined here.
+					AccountName:  "not used for now",
```

Comment:
Hi, @saragluna 
About the service name, we have 2 options here:
1. Keep current implementation: Use `${abbrs.xxx}${resourceToken}` for all services except the ACA (project name),
  1.1. Pros: It can help to avoid conflict.
  1.2. Cons: The name is not user-friendly, i.e. not human-readable.
2. Use customer defined name by analyzing customer's project or ask customer to input.
  2.1. Pros: The name is user-friendly.
  2.2. It can have conflict with existing resources.

I prefer option 1, because name conflict may bring customer trouble. What do you think?
If we choose option 1, we should delete all places that ask customer to input service names, for example: Azure Service Bus, Azure Event Hubs. The names are asked to input, but not used. Related screenshot:
> ![image](https://github.com/user-attachments/assets/d287850a-2a5b-429b-86d4-7586a9881ef5)

Reply:
Yes, I agree with you, we could use option 1.

Reply:
Then how about letting me delete `Azure dependency name:` related code in another PR?

Reply:
Yes, please